pupil_src/main.py

```diff
@@ -18,9 +18,22 @@
 
 import launchable_args
 
-default_args = {"app": "capture", "debug": False, "profile": False}
+default_args = {"app": "capture", "debug": False, "profile": False, "version": False}
 parsed_args, unknown_args = launchable_args.parse(running_from_bundle, **default_args)
 
+# app version
+from version_utils import get_version
+
+app_version = get_version()
+if parsed_args.version:
+    running_from = "bundle" if running_from_bundle else "source"
+    version_message = (
+        f"Pupil {parsed_args.app.capitalize()} version {app_version} ({running_from})"
+    )
+
+    print(version_message)
```

Comment:
shouldn't we `logger.info` this instead of `print`?

Reply:
The loggers are not setup yet at this point.